### ISSUE TITLE
When no patch-level updates are found, ncu reports it as an error

### ISSUE THREAD

**Issue by dangrussell** [raineorshine/npm-check-updates#944]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce
Run this command in a project with no minor updates available:
`ncu --target patch`

Error:
`No upgrades were returned. This is likely a problem with your installed yarn, the npm registry, or your Internet connection. Make sure npx pacote packument ncu-test-v2 is working before reporting an issue.`

I am using the latest Yarn (1.22.11), if that matters.

This is a new issue since the latest ncu release.

## Current Behavior
Reports a problem indicating a potential error, and suggests running a test.

FWIW, here's my result from running the suggested test (`npx pacote packument ncu-test-v2`)
```
{
  name: 'ncu-test-v2',
  'dist-tags': { latest: '2.0.0' },
  versions: {
    '0.1.0': { name: 'ncu-test-v2', version: '0.1.0', dist: [Object] },
    '1.0.0': { name: 'ncu-test-v2', version: '1.0.0', dist: [Object] },
    '2.0.0': { name: 'ncu-test-v2', version: '2.0.0', dist: [Object] }
  },
  modified: '2020-04-27T21:48:14.037Z',
  _cached: true,
  _contentLength: 3824
}
```

## Expected Behavior
Should simply state there are no updates.

**Comment by kevinwang89** [comment]
encountered same problem with newest version 11.8.4

**Comment by Zamiell** [comment]
I use NCU every day and this started happening yesterday.

**Comment by raineorshine** [comment]
Thanks for reporting! I found the problem. Should be fixed in `v11.8.5`.

**Comment by Zamiell** [comment]
Looks fixed now, thank you very much. <3
